crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -80,13 +87,32 @@ impl Auction {
                             fee: bigdecimal::BigDecimal::new(3.into(), 3),
                         })
                     }
-                    liquidity::Kind::UniswapV3(_) => todo!(),
+                    liquidity::Kind::UniswapV3(pool) => {
+                        tokens.insert(pool.tokens.get().0.into(), Default::default());
+                        tokens.insert(pool.tokens.get().1.into(), Default::default());
```

Comment:
Doesn't this overwrite existing and correct token metadata with empty token metadata?

Reply:
Is that expected? This was a bit of a surprise to me.

Reply:
No, it's not intended. 👍 was supposed to mean you found a legitimate issue. Will address it shortly.

Reply:
added: [b2c52d0](https://github.com/cowprotocol/services/pull/1251/commits/b2c52d07731863536b0df6a649c4fc5cbfbfa0c1)